servers/xr/xr_hand_tracker.cpp

```diff
@@ -110,48 +103,11 @@ void XRHandTracker::set_tracker_type(XRServer::TrackerType p_type) {
 }
 
 void XRHandTracker::set_tracker_hand(const XRPositionalTracker::TrackerHand p_hand) {
-	ERR_FAIL_INDEX(p_hand, TRACKER_HAND_MAX);
-
-	switch (p_hand) {
-		case TRACKER_HAND_LEFT:
-			tracker_hand = TRACKER_HAND_LEFT;
-			hand = HAND_LEFT;
-			break;
-
-		case TRACKER_HAND_RIGHT:
-			tracker_hand = TRACKER_HAND_RIGHT;
-			hand = HAND_RIGHT;
-			break;
-
-		case TRACKER_HAND_UNKNOWN:
-		default:
-			ERR_FAIL_MSG("XRHandTracker must specify hand");
-			break;
+	if (p_hand != TRACKER_HAND_LEFT && p_hand != TRACKER_HAND_RIGHT) {
```

Comment:
Maybe use `ERR_FAIL_COND_MSG` here?

Reply:
Good suggestion - updated to use ERR_FAIL_COND_MSG.